Add a section for scalar vs. timeseries in wildcard widget (#26324)
DataDog/documentation: content/en/dashboards/widgets/wildcard.md

1. [Check native widgets][4]. See if a Datadog widget can fulfill your requirements.
1. If no Datadog widget meets your requirements, in a new or pre-existing dashboard, click **Add Widgets**.
1. Click and drag the Wildcard Widget icon from the widget tray. 

> 1. Select from the **Request Type** dropdown. For more information on Scalar and Timeseries types, see the [Formulas Scalar vs. Formulas Timeseries] section of this page.

Author: ```suggestion
1. Select from the **Request Type** dropdown. For more information on Scalar and Timeseries types, see the [Formulas Scalar vs. Formulas Timeseries](#formulas-scalar-vs-formulas-timeseries) section of this page.
```

1. (Optional) Debug Vega-Lite specification mismatches with [Data Preview](#data-preview). Make sure the query in your Vega-Lite specification maps to the Datadog query. 
1. Click **Save**.

#### Formulas Scalar vs. Formulas Timeseries

> In Datadog dashboards, scalar and timeseries visualizations serve different purposes and are used to represent data in distinct ways.

Reviewer: `Visualization` might not be the noun we're looking for, we  are referring to a data format (which has a corresponding "request type". Maybe something like

> visualizations are powered by a multiple _request types_, including scalar and timeseries. Each _request types_ changes the number and type of fields available for the data in a Wildcard Widget.

There are 2 other request types in the widget selector which are GA, List and Histogram. It's OK to not explain what all 4 of them in 1 PR, but we don't want people to have the impression that there are only 2 request types.

**Timeseries**

> : This visualization is designed to display how your data changes over time.

Author: ```suggestion
: This data format is designed to display how your data changes over time.
```

   - **Use-cases**: It's ideal for monitoring metrics that fluctuate, such as CPU usage, memory consumption, or request rates. It helps identify trends, patterns, and anomalies over a specified time range. 

>    - **Appearance**: These visualizations are typically line or area charts. They show datapoints plotted against time (usually on the x-axis), to observe changes and trends.

Reviewer: I'm not sure we need to keep this section (since we are trying to focus convey that the request type is something that is independent of the visualization type - timeseries queries also can be used in the Heatmap widget), but if we do, DD timeseries are drawn in bar mode too (likely more frequently than areas).

>    - **Appearance**: These visualizations are typically line or area charts. They show data points plotted against time (usually on the x-axis), making it easy to observe changes and trends.

Author: ```suggestion
   - **Appearance**: These visualizations are typically line or area charts. They show datapoints plotted against time (usually on the x-axis), to observe changes and trends.
```

**Scalar**

> : This visualization presents a single, aggregate value derived from your data.

Reviewer: This describes the "query value" case, but doesn't fully describe how the scalar endpoint is used. The scalar endpoint powers the toplist, treemap, sumburst, and table widgets because 1 value is produced per `groupBy` and per `formula` .

I think that referring to these two sections as describing "data formats" rather than "visualization types" may make this easier to follow.

I think the takeaway that "if you need to describe a change over time, use timeseries, otherwise prefer scalar" works well for this section. 

We may also want to touch briefly on why "Formulas" appears in the title. The other two request formats (histogram, list), are not compatible with [Functions](https://docs.datadoghq.com/dashboards/functions/) API (also called "Formulas and Functions" / "F+F" internally), which is why only Scalar/Timeseries have the formulas prefix.

> : This visualization presents a single, aggregate value derived from your data.

Author: ```suggestion
: This data format presents a single, aggregate value derived from your data.
```

>    - **Use-cases**: It's best for displaying key performance indicators (KPIs) or summary statistics such as averages, sums, or percentiles. It provides a summary view of the current state or a specific metric.

Author: ```suggestion
   - **Use-cases**: It's best for displaying key performance indicators (KPIs) or summary statistics such as averages, sums, or percentiles. It provides a summary view of the current state or a specific metric. If you are not describing changes over time, use Scalar.
```

   - **Appearance**: Scalars often appear as large, standalone numbers or small cards on dashboards, highlighting the most relevant statistics, like the current number of active users, or the sum of errors over the last hour.

> Timeseries visualizations emphasize data trends over time, scalar visualizations focus on presenting single, computed values that help with quick assessments. If you're going to visualize time on an axis, or need individual time buckets, choose the Timeseries type. However, if you're not vizualizing against time, choose the Scalar type as it's much more performant.

Author: ```suggestion
The Timeseries data format emphasizes data trends over time, while the Scalar format focuses on presenting single, computed values for quick assessments. Choose the Timeseries type if you need to visualize time on an axis or require individual time buckets. If not visualizing against time, select the Scalar type for performance. 

**Note**: The "Formulas" prefix is used specifically for Scalar and Timeseries formats because they are compatible with the Functions API. The other formats, such as Histogram and List, do not support this API.
```
